wolfcrypt/src/evp.c

```diff
@@ -6878,6 +6878,446 @@ int wolfSSL_EVP_get_hashinfo(const WOLFSSL_EVP_MD* evp,
 }
 #endif /* !defined(NO_PWDBASED) */
 
+/* Base64 encoding APIs */
+#if defined(WOLFSSL_BASE64_ENCODE) || defined(WOLFSSL_BASE64_DECODE)
+static struct WOLFSSL_EVP_ENCODE_CTX* wolfSSL_EVP_ENCODE_CTX_new_ex(void* heap);
+/*  wolfSSL_EVP_ENCODE_CTX_new allocates WOLFSSL_EVP_ENCODE_CTX
+ *  Returns WOLFSSL_EVP_ENCODE_CTX structure on success, NULL on failure.
+ */
+struct WOLFSSL_EVP_ENCODE_CTX* wolfSSL_EVP_ENCODE_CTX_new(void)
+{
+    return wolfSSL_EVP_ENCODE_CTX_new_ex(NULL);
+}
+/* wolfSSL_EVP_ENCODE_CTX_new_ex is a helper function of
+ * wolfSSL_EVP_ENCODE_CTX_new to allocate WOLFSSL_EVP_ENCODE_CTX structure.
+ */
+static struct WOLFSSL_EVP_ENCODE_CTX* wolfSSL_EVP_ENCODE_CTX_new_ex(void* heap)
```

Comment:
What was the reasoning for this function to return a "static" struct pointer?

Reply:
"static" was added to declare the function as "static function". However, the helper function is not worth much existence.
So removed it.